CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to Wallet-Provider
+Thank you for considering contributing to Wallet-Provider!
+
+## Reporting Bugs
+If you discover a bug, please create a [new issue](https://github.com/massalabs/wallet-provider/issues/new?assignees=&labels=issue%3Abug&template=bug.md&title=) on our GitHub repository.
+In your issue, please include a clear and concise description of the bug, any relevant code snippets, error messages, and steps to reproduce the issue.
+
+## Installation
+To start developing with wallet-provider, you must install all the necessary dev dependencies. You can do so by running the following command:
+
+```sh
+npm install
+```
+
+This will install all the required packages listed in the package.json file, allowing you to update, fix, or improve wallet-provider in any way you see fit. 
+
```

Comment:
You should also talk about the installation of playwright and its dependencies.

Reply:
Done